apps/src/studio/studio.js

```diff
@@ -2495,7 +2497,7 @@ Studio.reset = function (first) {
   };
 
   // Reset the record of the last direction that the user moved the sprite.
-  Studio.lastMoveSingleDir = null;
```

Comment:
Yep!  I changed it to start facing the same direction as BB-8 (east).  I'm using it for `moveForward` / `turnRight` blocks.